src/pages/EditRequestPage.js

```diff
@@ -101,6 +110,9 @@ function EditRequestPage({report, route, parentReport, policyCategories, policyT
     // A flag for showing the tags page
     const shouldShowTags = isPolicyExpenseChat && (transactionTag || OptionsListUtils.hasEnabledOptions(lodashValues(policyTagList)));
 
+    // A flag for showing tax rate
+    const shouldShowTax = isPolicyExpenseChat && policy.isTaxTrackingEnabled;
```

Comment:
We have shouldShowTax at two places. Can we put the logic in function

Reply:
Yes, will handle this